keras/ops/image_test.py

```diff
@@ -13,6 +13,11 @@
 
 
 class ImageOpsDynamicShapeTest(testing.TestCase):
```

Comment:
We should also test numerical correctness, in both channels_first and last format, for batched and unbatched.